pkg/testcase/secretsencrypt.go

```diff
@@ -0,0 +1,143 @@
+package testcase
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/rancher/distros-test-framework/shared"
+
+	. "github.com/onsi/gomega"
+)
+
+func TestSecretsEncrypt() {
```

Comment:
Will this work if we have a setup without split mode?

Reply:
fixed

Reply:
resolved from my perspective but I'll let @mdrahman-suse mark resolved if he feels the same